lib/TestRunner.rb

```diff
@@ -295,6 +294,7 @@ def do_run(profiling_stage)
     file.puts("sh -c #{ENV["TEST_DIR"]}/bin/target-run-test ; c=$? ; echo END_RUN ; if [ $c == 0 ] ; then echo OVERALL_STATUS=OK ; else echo OVERALL_STATUS=ERROR ; fi")
     file.puts("echo -n '=== Finish (test board time): ' &&  date")
     file.puts("pgid=`ps -ejH | grep -m 1 $pid | awk '{ print $2 }'` && (flock 105 ; kill -9 -$pgid) 105>1.lock") if @timeout > 0
+    file.flush
```

Comment:
This is unclear. `flush` is usually called in `close`. What is the reason to force `flush`?

Reply:
OK. Thanks.  File.flush is not needed here.